ports/stm/system_stm32h7xx.c

```diff
@@ -0,0 +1,425 @@
+/*
+ * Taken from ST Cube library and modified.  See below for original header.
```

Comment:
If these are taken because of the issue of making the whole thing a submodule, it would be good to label these with the commit of the original (assuming it's in github), so we can track changes.

Reply:
They need to be modified with custom bootloader code and other stuff too - they aren't from a git submodule but are generated by CubeMX, so I don't have a specific commit to point to.

Reply:
Thanks, got it.